package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "testcafe-browser-provider-saucelabs",
+  "version": "1.0.0",
+  "description": "saucelabs TestCafe browser provider plugin.",
+  "repository": "https://github.com/DevExpress/testcafe-browser-provider-saucelabs",
+  "homepage": "https://github.com/DevExpress/testcafe-browser-provider-saucelabs",
+  "author": {
+    "name": "Developer Express Inc.",
+    "url": "https://devexpress.com"
+  },
+  "main": "lib/index",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "test": "gulp travis",
+    "publish-please": "publish-please",
+    "prepublish": "publish-please guard"
+  },
+  "keywords": [
+    "testcafe",
+    "browser provider",
+    "plugin"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "babel-runtime": "^6.11.6",
+    "desired-capabilities": "^0.1.0",
+    "lodash": "^4.14.2",
+    "pify": "^2.3.0",
+    "pinkie": "^2.0.4",
+    "request": "^2.74.0",
+    "saucelabs-connector": "^0.2.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^6.1.2",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-runtime": "^6.12.0",
+    "babel-preset-es2015": "^6.13.2",
+    "babel-preset-es2015-loose": "^7.0.0",
+    "babel-preset-stage-3": "^6.11.0",
+    "chai": "^3.5.0",
+    "del": "^2.0.0",
+    "gulp": "^3.9.0",
+    "gulp-babel": "^6.1.2",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^3.0.1",
+    "node-version": "^1.0.0",
+    "publish-please": "^2.1.4",
+    "testcafe": "https://github.com/AndreyBelym/testcafe/releases/download/branch-browser-providers-aftermath2/testcafe-0.2.3-alpha.tgz",
```

Comment:
Provider already updated to reflect changes from https://github.com/DevExpress/testcafe/pull/727